features/promotion/managing_catalog_promotions/adding_catalog_promotion_with_fixed_discount_where_amount_is_decimal_number_with_comma.feature

```diff
@@ -0,0 +1,20 @@
+@managing_catalog_promotions
+Feature: Adding a new catalog promotion with fixed discount where amount is a decimal number with comma
```

Comment:
This scenario could be added as an additional one to the feature for creating catalog promotions as the purpose is IMO the same, there is no need to create a separate feature file for that.